src/main/java/com/fasterxml/jackson/core/io/IOContext.java

```diff
@@ -199,6 +233,16 @@ public StreamWriteConstraints streamWriteConstraints() {
         return _streamWriteConstraints;
     }
 
+    /**
+     * Returns : {@link ErrorReportConfiguration}, container for configuration values used when 
+     * handling errorneous token inputs. 
+     *
+     * @since 2.16
+     */
+    public ErrorReportConfiguration errorReportConfiguration() {
+        return _errorReportConfiguration;
```

Comment:
can this be called `getErrorReportConfiguration`? Jackson typically uses set/get method naming.